### ISSUE TITLE
Staging Review finding: Single radio button doesn't allow for user to change their response.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#86825]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** VYE Team
**Product:** Verify Your Enrollment
**Feature:** Verify Your Enrollment

## Findings details
**VA.gov Experience Standard - issue:** [User can't review, edit, or confirm their information before form submission.](https://design.va.gov/about/experience-standards/#user-can-review-edit-or-confirm-their-information-before-form-submission)
**VA.gov Experience Standard - category:** [Credibility](https://design.va.gov/about/experience-standards/#category-credibility)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/mgib-enrollments/verify-information/](https://staging.va.gov/education/verify-school-enrollment/mgib-enrollments/verify-information/), the "Yes, this information is correct" form input is a radio button rather than a checkbox.

[WCAG success criteria 3.3.4](https://www.w3.org/WAI/WCAG22/Understanding/error-prevention-legal-financial-data.html) requires that users have a mechanism to correct errors prior to submission. Since there is only one radio button in the fieldset, it's not possible to change your selection once you've made it. This is not an appropriate use of a radio button.

<details><summary>Link, screenshot or steps to recreate</summary>

!["Yes, this information is correct" radio button unselected, followed by legal text, a back button, and a disabled gray submit button.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/Brian-67530-2.png?raw=true)

</details>

## Recommended action

Use the `va-checkbox` component instead. Checkboxes allow users to toggle their response.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 3.3.4 Error Prevention (Legal, Financial, Data) - Level AA](https://www.w3.org/TR/WCAG21/#error-prevention-legal-financial-data)
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by wafimohamed** [comment]
changed Radio button to Checkbox https://github.com/department-of-veterans-affairs/vets-website/pull/30549

**Comment by jsimonVA** [comment]
Update has been made and is in staging.
